packages/cluster-ui/src/statementsTable/statementsTable.tsx

```diff
@@ -34,33 +37,43 @@ function makeCommonColumns(
       label: cx("statements-table__col-count--bar-chart__label"),
     },
   });
-  const retryBar = retryBarChart(statements, {
+  const rowsReadBar = rowsReadBarChart(statements, {
     classes: {
-      root: cx("statements-table__col-retries--bar-chart"),
-      label: cx("statements-table__col-retries--bar-chart__label"),
+      root: cx("statements-table__col-rows-read--bar-chart"),
+      label: cx("statements-table__col-rows-read--bar-chart__label"),
     },
   });
-  const rowsBar = rowsBarChart(statements, {
+  const bytesReadBar = bytesReadBarChart(statements, {
     classes: {
-      root: cx("statements-table__col-rows--bar-chart"),
-      label: cx("statements-table__col-rows--bar-chart__label"),
+      root: cx("statements-table__col-bytes-read--bar-chart"),
+      label: cx("statements-table__col-bytes-read--bar-chart__label"),
     },
   });
   const latencyBar = latencyBarChart(statements, {
     classes: {
       root: cx("statements-table__col-latency--bar-chart"),
     },
   });
+  const maxMemUsageBar = maxMemUsageBarChart(statements, {
```

Comment:
Could we make the new column layout configurable via `makeCommonColumns` and a prop on `StatementsPage`? This component is now shipped to CockroachCloud so needs to remain backwards compatible with 20.2 at the very least.